src/components/calcite-tabs/calcite-tabs.scss

```diff
@@ -21,6 +21,13 @@
   --calcite-tabs-tab-margin-end: 1.25rem;
 }
 
+:host > div {
```

Comment:
If this is to target anything that goes in the `<slot>`, should we consider using a universal selector (for a use case where the content inside the `<slot>` is something other than a div?)

Reply:
Ah, this is meant to target the `div` that's a direct child of the `calcite-tabs` element, the one that holds the `tab-nav` slot and the `section` element.

Reply:
Ah, misunderstood. 👍. I know that was there already but kind of weird we aren't just using the `:host` or at least naming that div 🤷‍♂ .

Reply:
That's a good point.  Just checked around and the div didn't seem to be needed anywhere, so I just removed it and placed the style on the `:host`.